safe_transaction_service/safe_messages/utils.py

```diff
@@ -1,22 +1,36 @@
 from typing import Any
 
-from eth_account.messages import defunct_hash_message
+from eth_account.messages import encode_defunct
 from eth_typing import ChecksumAddress, Hash32
 from safe_eth.eth import get_auto_ethereum_client
-from safe_eth.eth.eip712 import eip712_encode_hash
+from safe_eth.eth.eip712 import eip712_encode
 from safe_eth.safe import Safe
 
 
-def get_hash_for_message(message: str | dict[str, Any]) -> Hash32:
+def encode_eip191_message(message: str) -> bytes:
+    signable_message = encode_defunct(text=message)
     return (
-        defunct_hash_message(text=message)
+        b"\x19"
+        + signable_message.version
+        + signable_message.header
+        + signable_message.body
+    )
+
+
+def encode_eip712_message(message: dict[str, Any]) -> bytes:
+    return b"".join(eip712_encode(message))
+
+
+def get_message_encoded(message: str | dict[str, Any]) -> bytes:
```

Comment:
why does a message require a certain encoding? this is the function for the raw message -> bytes message encoding, right? (not the SafeMessage EIP712 struct)

Reply:
This is only to get an dictionary or a string and encode it (using either EIP712 or EIP191), but it's not the SafeMessage struct